### PR TITLE
Image metadata for basic profile

### PR DESCRIPTION
**Pull request by PaulLeCam**
As discussed with @oed, this would replace the need for a custom IPFS-based solution to support multiple image formats, instead allowing to reference multiple sources directly in the basic profile schema.

### CODE REVIEW

File: CIPs/CIP-19/CIP-19.md [hunk 1]
```diff
@@ -74,6 +74,43 @@ The Basic Profile schema defines the format of a document that contains the prop
...
+        },
+        "width": {
+          "$ref": "#/definitions/positiveInteger"
+        },
+        "height": {
+          "$ref": "#/definitions/positiveInteger"
+        },
+        "size": {
+          "$ref": "#/definitions/positiveInteger"
+        }
```
Comment (oed) [inline CIPs/CIP-19/CIP-19.md:96]: Is this the file size? Do we need that?

**Review by oed — APPROVED** [review]
Seems good. No mimetype?

File: CIPs/CIP-19/CIP-19.md (hunk 1 above)
Comment (PaulLeCam) [reply CIPs/CIP-19/CIP-19.md:100]: It's not a required field but could be useful for consumer, for example when loading over mobile networks.

**Comment by PaulLeCam** [comment]
I forgot about the mimeType, thanks for noticing! I just added it.

**Pull request merged**